src/tests/index.js

```diff
@@ -1,40 +1,42 @@
-import chai, { assert } from 'chai';
-import chaiHttp from 'chai-http';
```

Comment:
rename your file depending oon task you have

Reply:
I think it would break the testing configurations somehow, I didn't create the file I just added my tests according to my task